sos/plugins/saltmaster.py

```diff
@@ -0,0 +1,38 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class SaltMaster(Plugin, RedHatPlugin, UbuntuPlugin):
+    """Salt Master
+    """
+
+    plugin_name = 'saltmaster'
+    profiles = ('storage')
+
+    option_list = [
+        ("log", "gathers all salt master logs", "slow", False)
+    ]
+
+    packages = (
+        'salt-master',
+    )
+
+    def setup(self):
+        self.add_cmd_output([
+            "salt-key --list all",
+        ])
```

Comment:
No need for a list if passing a single command - the `add_cmd_output()` call will accept either a list of strings, or a single string (so you should be able to get it all on one line).